packages/http-client/src/remote-index-api.ts

```diff
@@ -16,11 +16,15 @@ export class RemoteIndexApi implements IndexApi {
   private readonly _collectionURL: URL
 
   constructor(apiUrl: URL) {
-    this._collectionURL = new URL('./collection', apiUrl)
+    this._collectionURL = new URL('./collection/', apiUrl)
   }
 
   async count(query: BaseQuery): Promise<number> {
-    throw new Error('Not implemented')
+    const queryURL = new URL('./count', this._collectionURL)
+    for (const key in query) {
+      queryURL.searchParams.set(key, query[key])
```

Comment:
this won't actually serialize the filter right.  You'll need to use the `serializeObjectToSearchParams` function that I added in https://github.com/ceramicnetwork/js-ceramic/pull/2455

Reply:
Quite probable. As we discussed, you merge #2455, then I merge this one or rebase, so that we share the same function to handle the filters.